options/options.html

```
@@ -35,6 +35,21 @@
             ></span>
             <span class="label">一般設定</span>
           </button>
+          <button
+            id="tab-advanced"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)